zokrates_cli/src/helpers.rs

```diff
@@ -0,0 +1,96 @@
+use core::convert::TryFrom;
+
+use crate::constants::*;
+
+pub enum Curve {
+    Bn128,
+    Bls12,
+}
+
+pub enum Backend {
+    Bellman,
+    #[cfg(feature = "libsnark")]
+    Libsnark,
+}
+
+pub enum ProvingScheme {
+    G16,
+    #[cfg(feature = "libsnark")]
```

Comment:
I see why this is like this but ideally we would want to decouple backend and proving scheme and keep all backends in there.